apps/web/src/components/Form.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useEffect, useRef, useState} from 'react';
+import { Stepper, Button, FormContent }  from '@components';
+
+export const Form = () => {
+  const [currentStep, setCurrentStep] = useState(1);
+  const isFirstStep = currentStep === 1;
+
+  useEffect(()=> {
+    
+  }, [currentStep]);
```

Comment:
Could we remove this line if it's not used

Reply:
Removed .